src/MutantRunResultMatcher.ts

```diff
@@ -1,5 +1,6 @@
 import Mutant from './Mutant';
 import {RunResult, Location} from './api/test_runner';
+import {CoverageResult} from './api/test_runner/Coverage';
```

Comment:
Yes, we should pass it through the `test_runner.ts` file. That way, we can later move around the files within the test_runner folder as long as we don't break the api (the api being the `api/test_runner`). The reason it is not yet in the file is probably laziness The import would look like this:

```typescript
import {CoverageResult} from './api/test_runner';
```